src/app/WCNInfo.tsx

```diff
@@ -0,0 +1,18 @@
+
+export interface WCNInfoProps {
+    email: string;
+    linkedIn: string;
+    year: number;
+}
+
+export const WCNInfo: WCNInfoProps = {
+    email: "womencodersnetwork@gmail.com",
+    linkedIn: "https://www.linkedin.com/company/womencodersnet/",
+    year: new Date().getFullYear(),
+};
+
+
```

Comment:
can you remove these extra lines and just keep one empty line at the end on line 13?

Reply:
@sunsplat updated the codes for both of the PR